### ISSUE TITLE
Run wct tests in UI-less OS

### ISSUE THREAD

**Issue by kumarpratyush** [Polymer/web-component-tester#540]
The prod and staging environment for us are deployed on a box which has UI-less OS (only server component). 
Basically, it does not have any browsers installed. 
Is there a way to trigger tests on such an environment. Right now, all my tests are failing saying chrome installation not found.

**Comment by mg80** [comment]
Take a look at [http://elementalselenium.com/tips/38-headless](url)

We're using Xvfb via:
`xvfb-run npm test`

**Comment by karolchmist** [comment]
Or take a look at https://github.com/t2ym/wct-headless

**Comment by mg80** [comment]
I agree.  I've switched to headless Chrome ever since Chrome 59 and it's worked great.  xvfb would really only make sense if the browser didn't support running headless.
